image/Makefile

```diff
@@ -18,34 +29,30 @@ help: ## show make targets
 .PHONY: clean
 clean: ## remove build artifacts
 	-$(RM) $(ENGINE_DIR)/Dockerfile.engine
-	-docker rmi $(DOCKER_HUB_ORG)/$(ENGINE_IMAGE):$(STATIC_VERSION)
-	-rm -f image-linux
+	-$(RM) $(ENGINE_DIR)/Dockerfile.engine-dm
+	-docker rmi $(IMAGE_WITH_TAG)
+	-rm -f image-linux-*
 
 .PHONY: image
 image: image-linux
 
-
-$(ENGINE_DIR)/Dockerfile.engine:
-	cp Dockerfile.engine $(ENGINE_DIR)
+$(ENGINE_DIR)/Dockerfile.%: Dockerfile.%
+	cp $< $@
 
 # builds across multiple archs because the base images
 # utilize manifests
 image-linux: $(ENGINE_DIR)/Dockerfile.engine
```

Comment:
As an alternative, maybe this target could be restructured to build both by defining a new target for the overlay image and one for dm?
```
image-linux: image-linux-o2 image-linux-dm

image-linux-o2: $(ENGINE_DIR)/Dockerfile.engine
        ...
```

Reply:
Hmm... now that I'm looking I can't seem to find where I saw it, but I swear somewhere in the CI pipeline scripts I saw a "make image-linux" (or something along those lines)